synadm/cli/user.py

```diff
@@ -359,17 +359,17 @@ def name_extra(self):
     of all rooms and deletes third-party identifiers (to prevent the user
     requesting a password reset). See also "user deactivate" command.""")
 @optgroup.option(
-    "--user-type", "user_type", type=str,
-    help="""Change the type of the user. Currently allowed values are 'bot'
-    and 'support'.""")
-@optgroup.option(
-    "--clear-user-type", "clear_user_type", is_flag=True,
-    help="""Clear the user_type field of the user.""")
+    "--user-type", type=str, default=None, show_default=True,
+    help="""Change the type of the user. Currently understood by the Admin API
+    are 'bot' and 'support'. Use 'regular' to create a regular Matrix user
+    (which effectively sets the user-type to 'null'). If the --user-type option
+    is omitted when modifying an existing user, the user-type will not be
+    manipulated. If the --user-type option is omitted when creating a new user,
```

Comment:
> If the --user-type option is omitted when creating a new user

But this command is about modifying, not creating, right?

Reply:
Nope, it's both! See admin api docs, there is one api for both creating and modifying.

Yeah an alias name for `modify` to `create` would be great! I want that since ages...